schema.graphql

```diff
@@ -50,9 +51,8 @@ type BookingNode implements Node {
   priceBreakdown: PriceBreakdownNode
   transactions(offset: Int, before: String, after: String, first: Int, last: Int, type: String, provider: String, createdAt: DateTime, id: ID): TransactionNodeConnection
   expired: Boolean!
-  totalSales: Int
-  totalRefunds: Int
-  netIncome: Int
+  salesBreakdown: SalesBreakdownNode
```

Comment:
So at the moment this will allow the user to query some information we might not want them to (like society transfer fee?) Can we somehow restrict based on permissions?